.github/workflows/ci.yml

```diff
@@ -152,6 +152,8 @@ jobs:
         run: yarn ci:test:plugin-web
 
   test-playwright:
+    environment: plugin-development
```

Comment:
@jonathanyeung the test-playwright job https://github.com/dendronhq/dendron/actions/runs/3638127632 failes with `Branch "chore/web-performance-harness" is not allowed to deploy to plugin-development due to environment protection rules.` Could you check what the cause of this is.

Reply:
It's because of this line in the workflow: 

```yaml
  test-playwright:
    environment: plugin-development
```

Looks like we need the secrets to do the airtable upload?  Environments need to have their branches whitelisted so that they can run (to protect secret access) - our secrets in plugin-development aren't very sensitive, so I can go ahead and enable access to the env for all branches